.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": true
```

Comment:
were we having discrepancies with our formatting?

Reply:
Oops, this shouldn't have been committed. I'll remove it.